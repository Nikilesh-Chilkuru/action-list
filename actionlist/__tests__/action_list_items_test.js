import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Ali from '../action_list_items';

test('renders correctly', () => {
  const tree = renderer.create(
    <Ali />,
  ).toJSON();

  expect(tree).toMatchSnapshot()
});
