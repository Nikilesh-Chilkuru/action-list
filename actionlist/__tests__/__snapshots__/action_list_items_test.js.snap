exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffffff",
    }
  }>
  <ScrollView
    dataSource={
      ListViewDataSource {
        "items": 5,
      }
    }
    initialListSize={10}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={1000}
    onKeyboardDidHide={undefined}
    onKeyboardDidShow={undefined}
    onKeyboardWillHide={undefined}
    onKeyboardWillShow={undefined}
    onLayout={[Function]}
    onScroll={[Function]}
    pageSize={1}
    removeClippedSubviews={true}
    renderRow={[Function]}
    scrollEventThrottle={50}
    scrollRenderAheadDistance={1000}
    stickyHeaderIndices={Array []}>
    <View>
      <ScrollView
        style={
          Object {
            "backgroundColor": "#f4f7f9",
            "paddingTop": 0,
          }
        }>
        <View
          style={
            Object {
              "backgroundColor": "#e9e9e9",
              "borderBottomWidth": 0.5,
              "overflow": "hidden",
              "paddingRight": 10,
            }
          }>
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#7B1500",
                  "flex": 1,
                  "fontWeight": "bold",
                  "padding": 10,
                }
              }>
              Document 1
            </Text>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hitSlop={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  undefined,
                ]
              }
              testID={undefined}>
              <Image
                source={1}
                style={
                  Object {
                    "height": 20,
                    "marginTop": 10,
                    "width": 20,
                  }
                } />
            </View>
          </View>
        </View>
      </ScrollView>
    </View>
    <View>
      <ScrollView
        style={
          Object {
            "backgroundColor": "#f4f7f9",
            "paddingTop": 0,
          }
        }>
        <View
          style={
            Object {
              "backgroundColor": "#e9e9e9",
              "borderBottomWidth": 0.5,
              "overflow": "hidden",
              "paddingRight": 10,
            }
          }>
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#7B1500",
                  "flex": 1,
                  "fontWeight": "bold",
                  "padding": 10,
                }
              }>
              Document 2
            </Text>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hitSlop={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  undefined,
                ]
              }
              testID={undefined}>
              <Image
                source={1}
                style={
                  Object {
                    "height": 20,
                    "marginTop": 10,
                    "width": 20,
                  }
                } />
            </View>
          </View>
        </View>
      </ScrollView>
    </View>
    <View>
      <ScrollView
        style={
          Object {
            "backgroundColor": "#f4f7f9",
            "paddingTop": 0,
          }
        }>
        <View
          style={
            Object {
              "backgroundColor": "#e9e9e9",
              "borderBottomWidth": 0.5,
              "overflow": "hidden",
              "paddingRight": 10,
            }
          }>
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#7B1500",
                  "flex": 1,
                  "fontWeight": "bold",
                  "padding": 10,
                }
              }>
              Document 3
            </Text>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hitSlop={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  undefined,
                ]
              }
              testID={undefined}>
              <Image
                source={1}
                style={
                  Object {
                    "height": 20,
                    "marginTop": 10,
                    "width": 20,
                  }
                } />
            </View>
          </View>
        </View>
      </ScrollView>
    </View>
    <View>
      <ScrollView
        style={
          Object {
            "backgroundColor": "#f4f7f9",
            "paddingTop": 0,
          }
        }>
        <View
          style={
            Object {
              "backgroundColor": "#e9e9e9",
              "borderBottomWidth": 0.5,
              "overflow": "hidden",
              "paddingRight": 10,
            }
          }>
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#7B1500",
                  "flex": 1,
                  "fontWeight": "bold",
                  "padding": 10,
                }
              }>
              Document 4
            </Text>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hitSlop={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  undefined,
                ]
              }
              testID={undefined}>
              <Image
                source={1}
                style={
                  Object {
                    "height": 20,
                    "marginTop": 10,
                    "width": 20,
                  }
                } />
            </View>
          </View>
        </View>
      </ScrollView>
    </View>
    <View>
      <ScrollView
        style={
          Object {
            "backgroundColor": "#f4f7f9",
            "paddingTop": 0,
          }
        }>
        <View
          style={
            Object {
              "backgroundColor": "#e9e9e9",
              "borderBottomWidth": 0.5,
              "overflow": "hidden",
              "paddingRight": 10,
            }
          }>
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#7B1500",
                  "flex": 1,
                  "fontWeight": "bold",
                  "padding": 10,
                }
              }>
              Document 5
            </Text>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hitSlop={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  undefined,
                ]
              }
              testID={undefined}>
              <Image
                source={1}
                style={
                  Object {
                    "height": 20,
                    "marginTop": 10,
                    "width": 20,
                  }
                } />
            </View>
          </View>
        </View>
      </ScrollView>
    </View>
  </ScrollView>
</View>
`;
